react/OrderShipping.tsx

```diff
@@ -1,31 +1,202 @@
-import React, { createContext, ReactNode, useContext } from 'react'
-
+import React, {
+  createContext,
+  ReactNode,
+  useContext,
+  useCallback,
+  useEffect,
+  useRef,
+} from 'react'
+import { compose, graphql } from 'react-apollo'
+import { useOrderQueue } from 'vtex.order-manager/OrderQueue'
 import { useOrderForm } from 'vtex.order-manager/OrderForm'
 
+import {
+  estimateShipping as EstimateShipping,
+  selectDeliveryOption as SelectDeliveryOption,
+} from 'vtex.checkout-resources/Mutations'
+
 interface Context {
   countries: string[]
+  selectedAddress: CheckoutAddress
+  insertAddress: any
```

Comment:
Don't you have the typings for that?

Reply:
I kind of don't know how could I typify that. `insertAddress` is a function which triggers a graphql mutation.

Reply:
Let me know if you have a way of doing that, Lucis :)

Reply:
hey, @jeffersontuc, did a little search. Isn't it of `(CheckoutAddress) => void` type?

Reply:
It's, indeed! thanks :*